h2o-bindings/bin/custom/python/gen_stackedensemble.py

```diff
@@ -134,6 +134,12 @@ def extend_parms(parms):
 else:
     assert_is_type({pname}, None, [str])
     self._parms["{sname}"] = {pname}
+""",
+        getter="""
+base_models = self._parms.get("base_models")
+if base_models is None and self.actual_params.get("base_models"):
```

Comment:
I would reverse the logic (first check `actual_params`, and fallback to local `base_models` value): source of truth is always the backend, we never know what happens there...

Reply:
also, is `self.actual_params["base_models"]` returning a list of `ids` or a list of complex keys?
One of the goal of the ticket is to provide something directly usable, we want sth like:
```
> print(se.base_models)
> ['model_1', 'model_2', 'model_3', ...]
```

Reply:
Thanks for the suggestion. That makes much more sense.

Reply:
I understood from the Jira that we want the model IDs. I will have to check tomorrow if we really get them now. When I tested the previous version with the fallback to `actual_params` it returned in both cases the model IDs.

Reply:
I added assertion to the test to expect list of strings as base models so the test would fail in case of different implementation `actual_params` in future.